api/process.go

```diff
@@ -86,6 +86,38 @@ func (a *API) createProcessHandler(w http.ResponseWriter, r *http.Request) {
 		process.Address = processInfo.Address
 	}
 
+	// if it's a new draft process
+	if process.Address.Equals(nil) && process.ID == primitive.NilObjectID {
+		// Check if the organization has a subscription
+		org, err := a.db.Organization(process.OrgAddress)
+		if err != nil {
+			errors.ErrOrganizationNotFound.WithErr(err).Write(w)
+			return
+		}
+
+		if org.Subscription.PlanID == 0 {
+			errors.ErrNoOrganizationSubscription.With("can't create draft process").Write(w)
+			return
+		}
+
+		plan, err := a.db.Plan(org.Subscription.PlanID)
```

Comment:
Do you think it would make sense to have checks like these here:
https://github.com/vocdoni/saas-backend/blob/6f2569b756c7873c329e87c93edae20660d56bfb/subscriptions/subscriptions.go#L167

?